README.md

```diff
@@ -8,7 +8,7 @@ Example:
 GET
 ```java
 HttpRequest request = httpRequest(new TargetUrl("abv.bg")).build();
-ChitaHttp chitaClient = new ChitaHttpClient();// in most cases chitaClient should be injected
+ChitaHttp chitaClient = new HttpClient();// in most cases chitaClient should be injected
```

Comment:
Yes, you have a point.
Although I can't think right now of a way for testing code that uses chita, except with the [rest-driver](https://github.com/rest-driver/rest-driver) lib that you suggested a while back ?